libpod/events.go

```diff
@@ -105,9 +105,12 @@ func (c *Container) newContainerExitedEvent(exitCode int32) {
 	intExitCode := int(exitCode)
 	e.ContainerExitCode = &intExitCode
 
-	e.Details = events.Details{
-		Attributes: c.Labels(),
+	attrs := c.Labels()
+	if c.state.OOMKilled {
+		oomKilled := true
+		e.OOMKilled = &oomKilled
```

Comment:
Why is allocated new variable?
```suggestion
	if c.state.OOMKilled {
		e.OOMKilled = &c.state.OOMKilled
```